src/modules/Board/components/TaskFormDialog.tsx

```diff
@@ -148,7 +200,7 @@ export function TaskFormDialog({
       </DialogContent>
       <DialogActions>
         <Button onClick={() => dialog.close()}>Cancel</Button>
-        <Button onClick={() => dialog.close()}>
+        <Button onClick={() => saveForm()}>
```

Comment:
I guess react-hook-form expect using `Controller` component or moving buttons to the form for correct behavior. I mean it doesn't look correct to `trigger` and check `isValid` by yourself. react-hook-form should do it. But right now it's better to skip this fix